docs(llmq): v19 changes in DIPs (#122)
dashpay/dips: dip-0003.md

A ProRegTx is invalid if any of these conditions are true:

In the case of type 0:
  1. collateralOutpoint `hash` is null but an output with 1000 DASH is not present at position `n` of the ProRegTx outputs
  2. collateralOutpoint `hash` is not null but an output with 1000 DASH can't be found in the UTXO specified by the `hash` and `n`

> In the case of type 1:

Reviewer: I think you need a paragraph break. This list doesn't render as desired (see https://github.com/dashpay/dips/blob/bls_dips_adaptations/dip-0003.md#validation-rules)
```suggestion

In the case of type 1:
```

  1. collateralOutpoint `hash` is null but an output with 4000 DASH is not present at position `n` of the ProRegTx outputs
  2. collateralOutpoint `hash` is not null but an output with 4000 DASH can't be found in the UTXO specified by the `hash` and `n`

> Applicable to any type:

Reviewer: Ditto
```suggestion

Applicable to any type:
```

A ProRegTx is invalid if any of these conditions are true:

In the case of type 0:
  1. collateralOutpoint `hash` is null but an output with 1000 DASH is not present at position `n` of the ProRegTx outputs
  2. collateralOutpoint `hash` is not null but an output with 1000 DASH can't be found in the UTXO specified by the `hash` and `n`

In the case of type 1:
  1. collateralOutpoint `hash` is null but an output with 4000 DASH is not present at position `n` of the ProRegTx outputs
  2. collateralOutpoint `hash` is not null but an output with 4000 DASH can't be found in the UTXO specified by the `hash` and `n`

> Applicable to any type:

Reviewer: This one still isn't working if you look at the preview. Try adding a blank line / restarting numbering at 1.
```suggestion
Applicable to any type:

```

Author: @thephez Should be good now

Please note that while deploying DIP3, additional and temporary validation rules will apply. The details of these temporary rules will be described in the deployment plan.

### ProRegTx and ProUpServTx

A ProRegTx or ProUpServTx is invalid if any of these conditions are true (applicable only when type is 1):

  1. platformP2PPort, platformHTTPPort and port from ipAddress are not distinct.
  2. platformNodeID is null or is a non-hex string.
  3. platformNodeID is not unique in the network.

> 

Reviewer: ```suggestion
```

Please note that while deploying DIP3, additional and temporary validation rules will apply. The details of these temporary rules will be described in the deployment plan.

### ProRegTx and ProUpServTx

A ProRegTx or ProUpServTx is invalid if any of these conditions are true (applicable only when type is 1):

  1. platformP2PPort, platformHTTPPort and port from ipAddress are not distinct.

>   2. platformP2PPort, platformHTTPPort are not set to default mainnet values.

Reviewer: This may change though, correct?

Author: Correct.

The new rules to determine the next block’s payee are:

> Find previous Masternode payee. If the last payee is a Regular Masternode, go to step #5
> If the last HPMN payee was already paid 4 times, go to step #5
> If the last HPMN is not present in current MN list anymore, go to step #5
> Select last HPMN as payee and increment his consecutive payment state by one. Return
> 
> 1. Take the last block's payee. If its type is Regular Masternode, go to step 5.
> 2. If the last HPMN payee was already paid 4 blocks in a row, go to step 5.
> 3. If the last HPMN is not present in current Masternode list anymore, go to step 5.
> 4. Select last HPMN payee as the next block's payee. Return

Reviewer: Looks like there is some duplicate text.
```suggestion
1. Take the last block's payee. If it is a regular masternode (type 0), go to step 5.
2. If the last HPMN payee was already paid 4 blocks in a row, go to step 5.
3. If the last HPMN is not present in the current masternode list anymore, go to step 5.
4. Select last HPMN payee as the next block's payee. Return.
```
